conwayste/src/ui/widget.rs

```diff
@@ -68,6 +71,16 @@ pub trait Widget: Downcast {
         Ok(())
     }
 
+    /// Textfield gains focus and begins accepting user input
```

Comment:
Should say widget here and below. There could be more than one kind of thing that takes focus.